bitnami/apache/templates/deployment.yaml

```diff
@@ -90,8 +90,8 @@ spec:
           {{- if .Values.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
-              path: /
-              port: http
+              path: {{ .Values.readinessProbe.path }}
+              port: {{ .Values.readinessProbe.port }}
```

Comment:
I understand that you want to modify the "path" to check, but why would you change the port? I mean, no matter what port you chose at `.Values.service.port`, the port used in the container will be named "http" (check line 75)

Reply:
changed port back to http to use port value declared at line 73